### DISCUSSION TITLE
Controlling section numbering

### DISCUSSION THREAD

**Discussion by pfelber** [executablebooks/meta#113] [Q&A]
I probably overlooked it, but I could not find an easy way to controls the levels of section numbering. I'd like to have numbers for sections and subsections, but not below (i.e., 1, 1.1, and then the next level headers ### would not have numbers). I have specified "numbered: true" in _toc.yml and I use markdown.
Can I do that easily? Thanks!

**Comment by mmcky** [comment]
Hi @pfelber. To my knowledge the `numbered: true` is globally configured but one way to achieve what your after is by opting for local control of the section numbering. However this comes with a **tradeoff** as numbering would no longer be automatic and may be frustrating to update if your document is dynamic in terms of adding new sections / chapters etc. regularly. 

**Steps:**
1. turn off the global `numbered: true` setting
2. Add the following directive at the top of each file

````
```{sectnum}
:depth: 2
:start: {{ integer to start your numbering from }}
```
````

the `depth` would show numbers for `level=1` and `level=2`

This functionality is provided by an underlying software layer called `docutils` and the configurable options are [available here](https://docutils.sourceforge.io/docs/ref/rst/directives.html#automatic-section-numbering) for reference.

I have also setup a [sample project](https://github.com/mmcky/ebp-test-projectstructure/tree/master/parts_chapters/sectnum) that you could refer to if you did want to set this up or see how this might work.

We are also actively working on ways to improve the `_toc.yml` structure and supported features.
